cli/azd/pkg/apphost/generate_test.go

```diff
@@ -281,6 +281,14 @@ func TestAspireContainerGeneration(t *testing.T) {
 	m, err := ManifestFromAppHost(ctx, filepath.Join("testdata", "AspireDocker.AppHost.csproj"), mockCli, "")
 	require.NoError(t, err)
 
+	for _, name := range []string{"mysqlabstract", "my-sql-abstract", "noVolume"} {
```

Comment:
just a small nit: This feels like a test that wouldn't catch these types of issues, the baseline doesn't protect any further against regressions here, and the snapshot is too wide to be humanly verifiable IMO. We'd want a validation of the container app manifest against the well-known schema, I logged #3890.

Reply:
agree.  Those snapshot tests are super annoying!!! lol =)
OTOH, they are super simple, compared to something more elaborated.
I'd be happy to get rid of those.
The ultimate test here for volumes would be updating the functional test to deploy one app with volumes, write some data and then deploy the app again and test the data is persisted....  :)